specifications/service-registry.yaml

```diff
@@ -0,0 +1,279 @@
+openapi: 3.0.0
+info:
+  description: 
+    API example for service registry. The API provides methods of distributing
+    and searching metadata related to mobility services. The API provides a way
+    for the service providers to publish their services and the the MaaS-operators
+    and end users to query available services and the user API standards in any 
+    area of interest. The services can be filtered by used API specifications 
+    and the certification proviced by trusted third parties. The API also provides
+    methods for distributing the registered metadata between multiple instances
+    of the service.<br/>
+    <br/>
+    Copyright 2017 FLOU Ltd.
+  version: "0.1.0-oas3"
+  title: Service registry
+  contact:
+    name: FLOU
+    url: 'https://www.flou.io/'
+    email: sami.makinen@flou.io
+  license:
+    name: Apache License 2.0
+    url: 'https://github.com/maas-alliance/apis/LICENSE'
+tags:
+  - name: catalogs
+    description: Handle service catalog registration
+  - name: queries
+    description: Query registered services
+paths:
+  /catalogs/announce:
+    post:
+      tags:
+        - catalogs
+      summary: Add a new service catalog to the registry
+      description: 
+        Announce new service provider to be registered. The services area 
+        registered by providing a link to a service catalog file that specifies
+        the services and APIs provided by the service provides. The Service 
+        Registry will then update its internal database by fetching and parsing
+        the service catalog file.
+      operationId: announceCatalog
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                uri:
+                  description: Location of the service catalog resource to register
+                  type: string
+                  format: catalogUri
```

Comment:
It's unclear to me what exactly these URLs point at. What it be the base URL of someone implementing the `/services/find/` endpoint listed below?